src/pygambit/tests/test_game.py

```diff
@@ -74,6 +74,6 @@ def test_game_dereference_invalid():
     game = gbt.Game.new_tree()
     game.add_player("One")
     strategy = game.players[0].strategies[0]
-    game.append_move(game.root, game.players[0], ["a", "b"])
+    game.append_move([game.root], game.players[0], ["a", "b"])
```

Comment:
The original test is correct - that you had to change this is an indication that your code is wrong, because the spec states that the new implementation is backwards-compatible.

If a test fails, fix the code don't change the test.

This comment applies equally to a number of the other changes both to the tests and the User Guide - in particular everything in the User Guide should run unchanged.